### ISSUE TITLE
Changes on integrations settings not saved on Chrome

### ISSUE THREAD

**Issue by lukastmf** [clockify/browser-extension#265]
### Summary

The changes in the integrations settings are not saved. This has the effect that the clockify extension works on sites which are selected by default (i.e. google docs) but does not work on added custom domains URL's. Also deselection of default URL's are not present after restart of browser.

Also it seams the click on save has no effect.

### Steps to reproduce

1. Install clockify extension and log into your account
2. Go to option / integrations
3. Deselect an option (i.e. Cerb - cerb.ai)
4. Add a Custom domains
5. Click "Save"
6. Restart the browser

### Persons affected

All users not using default selection

### Platform and version

- MAC OS Monterey (12.7.4)
- Chrome 125.0.6422.112
- Clockify tracker Version 2.10.20

### What is the current *bug* behavior?

- Logging does not work on Custom domains
- Change in settings not present after chrome restart

### What is the expected *correct* behavior?

- Logging does work on Custom domains
- Change in settings still present after chrome restart

### Relevant logs and/or screenshots

before restart:
![image](https://github.com/clockify/browser-extension/assets/170924476/177aa5a5-1c43-4a51-be79-528e227b1a1f)

After restart
![image](https://github.com/clockify/browser-extension/assets/170924476/ebfa4054-f6ad-4c77-a776-abadfcd5f4ed)

**Comment by wiegandj** [comment]
We have the same or at least a very similar issue.

Custom domains don't save for a longer period of time.

This bug does not only occur when restarting the browser, but also gets triggered by something different before closing the browser. We use the integration for our own GitLab too and the custom domain gets removed after a short period of time. We have tested around a bit but have not found the trigger for the deletion yet.

### What is the current bug behavior?
Custom domains get deleted after a while for unknown reason.

### Since when is the bug happening?
- about 1.5 weeks

### What is the expected correct behavior?
Custom domains stay in the Clockify settings.

### Steps to reproduce
1. Install Clockify extension and log into your account
2. Go to option / integrations
3. Add a Custom domain
4. Click "Save changes"
5. After a while, the Custom domain is gone

### Persons affected
All users using a Custom domain.

### Platform and version
Linux Ubuntu (does not matter)
Version 125.0.6422.60
Clockify tracker Version 2.10.20

**Comment by kubaty007** [comment]
I have same problem on Chrome 125.0.6422.77. I have to add custom domain multiple times per day.

**Comment by timalenustmf** [comment]
same issue here with our self hosted gitlab

**Comment by lockHrt** [comment]
Yes, I like to disable Clockify integration on several websites as it slows down those websites (like Asana, monday.com etc). But the extension doesn't remember that settings for long. It auto enables integration for all websites.

**Comment by da-mask** [comment]
Another 'same here' comment. Attempting to add a custom domain resets all integration settings back to default, it seems.

**Comment by Yggwit** [comment]
Another 'same here' comment. I am loosing my config many times a day.
Many users from my company are having troubles too.
W10, Chrome 125.0.6422.141.

**Comment by mmaldonadodev** [comment]
I have same problem on Chrome 125.0.6422.76

**Comment by jdumaz** [comment]
Same issue on Firefox 126.0.1.

**Comment by Yggwit** [comment]
> Another 'same here' comment. I am loosing my config many times a day. Many users from my company are having troubles too. W10, Chrome 125.0.6422.141.

It looks fixed on W10, Chrome 125.0.6422.142.
No problem since last update.

**Comment by mmaldonadodev** [comment]
Well, it seems so, I have updated to the latest version of Chrome 125.0.6422.141, and it works until now

**Comment by CluelessCPU** [comment]
It looks like there's the same issue on the Firefox extension

**Comment by dmitrijkiltau** [comment]
Same here. Chrome 131.0.6778.69, Ubuntu. Out of nothing the settings get reset, sometimes it seems to be because of an update.
